prov/shm/src/smr_ep.c

```diff
@@ -1189,80 +1040,6 @@ void smr_ep_exchange_fds(struct smr_ep *ep, int64_t id)
 		SMR_CMAP_FAILED : SMR_CMAP_SUCCESS;
 }
 
-static void smr_init_ipc_socket(struct smr_ep *ep)
-{
-	struct smr_sock_name *sock_name;
-	struct sockaddr_un sockaddr = {0};
-	int ret;
-
-	ep->sock_info = calloc(1, sizeof(*ep->sock_info));
-	if (!ep->sock_info)
-		goto err_out;
-
-	ep->sock_info->listen_sock = socket(AF_UNIX, SOCK_STREAM, 0);
-	if (ep->sock_info->listen_sock < 0)
-		goto free;
-
-	snprintf(smr_sock_name(ep->region), SMR_SOCK_NAME_MAX,
-		 "%ld:%d", (long) ep->region->pid, ep->ep_idx);
-
-	sockaddr.sun_family = AF_UNIX;
-	snprintf(sockaddr.sun_path, SMR_SOCK_NAME_MAX,
-		 "%s%s", SMR_ZE_SOCK_PATH, smr_sock_name(ep->region));
```

Comment:
There are probably some defines you can remove - `SMR_SOCK_NAME_MAX` and `SMR_ZE_SOCK_PATH`